examples/oft-aptos/.gitignore

```diff
@@ -0,0 +1,6 @@
+.env
+*/build
+*/node_modules
+.aptos
+cache/hardhat
+deployments/*
```

Comment:
deployments folder shouldn't be in gitignore, it also isn't in other code examples we have in devtools please cross check